backend/app/usecase/match/contains_all.go

```diff
@@ -0,0 +1,25 @@
+package match
+
+import "strings"
+
+var _ Keyword = (*ContainsAll)(nil)
+
+// ContainsAll checks if an input contains all the elements of a list.
+type ContainsAll struct {
+}
+
+// IsMatch checks if the input contains all the words.
+func (c ContainsAll) IsMatch(words []string, input string) bool {
+	if input == "" {
```

Comment:
Shouldn't `IsMatch([""], "")` return `true`?

Reply:
I guess, we can remove this condition. Empty string is still a valid string and if I am not wrong, `strings.Contains(input, word)` should handle this edge case by itself.

Reply:
That is true, my bad. Thank you very much for pointing it out.